pandas/io/clipboards.py

```diff
@@ -121,14 +121,13 @@ def to_clipboard(obj, excel=True, sep=None, **kwargs):  # pragma: no cover
             # clipboard_set (pyperclip) expects unicode
             obj.to_csv(buf, sep=sep, encoding='utf-8', **kwargs)
             text = buf.getvalue()
-            if compat.PY2:
+            if PY2:
                 text = text.decode('utf-8')
             clipboard_set(text)
             return
         except TypeError:
             warnings.warn('to_clipboard in excel mode requires a single '
-                          'character separator. Set "excel=False" or change '
-                          'the separator')
```

Comment:
@david-liu-brattle-1 I removed the last sentence of this warning, as setting `excel=False` does not help as then you get the warning that the separator is ignored. It's simply that `to_clipboard` does not support multiple character separator at all.